bigtable/client/data_test.cc

```diff
@@ -1,1626 +0,0 @@
-// Copyright 2017 Google Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-#include "bigtable/client/data.h"
-
-#include <array>
-#include <iostream>
-#include <numeric>
-#include <sstream>
-
-#include <absl/memory/memory.h>
-#include <gmock/gmock.h>
-#include <google/bigtable/v2/bigtable_mock.grpc.pb.h>
-#include <google/protobuf/text_format.h>
-
-#include <bigtable/client/readrowsparser.h>
-
-using google::bigtable::v2::ReadRowsResponse;
-
-// TODO(dmahu): this is copied from "grpc++/test/mock_stream.h"
-// which is not installed by default by grpc.
-template <class R>
-class MockClientReader : public grpc::ClientReaderInterface<R> {
```

Comment:
Why is this entire file deleted in a PR addressing "adding issues to all TODO()s"? Is all this code auto-generated and is removed due to protobufs now supporting `std::move`? It's not obvious from the PR description how it follows that this file should be entirely deleted. Can you please clarify?

Reply:
This file (`bigtable/client/data_test.cc`) was not in use.  I think removing the file is a cleaner fix than adding a TODO and then removing the file in another PR.

And when we upgraded the grpc version in #86 we also got move constructors for protos, so the easiest fix for the TODOs about them was to use the default move operators instead of opening a separate bug and then closing it.

Reply:
It looks like the tests in this file are acceptance tests, converted from JSON in another repo:

> ```cpp
> // The tests included between the markers below are defined in the
> // file "read-rows-acceptance-test.json" in the cloud-bigtable-client
> // repository and formatted as C++ code using tools/convert_tests.py
> ```

Are you sure we should delete this file? Is there another test that does the same thing?

Reply:
The file was not compiled, nor are any of those tests executed anywhere.

Yes, we will need it (or a file like it), when we have an implementation of `ReadRows()`, but right now it is just not used.